src/citrine/resources/table_config.py

```diff
@@ -228,7 +232,11 @@ def add_all_ingredients(self, *,
             datasets=copy(self.datasets),
             rows=copy(self.rows),
             variables=copy(self.variables) + new_variables,
-            columns=copy(self.columns) + new_columns
+            columns=copy(self.columns) + new_columns,
+            config_uid=copy(self.config_uid),
+            version_uid=copy(self.version_uid),
```

Comment:
I don't think we want to copy the version_uid, since that is provided by the service when the config is registered.
```suggestion
```